core-models/src/main/scala/com/mesosphere/usi/core/linting/Linting.scala

```diff
@@ -0,0 +1,47 @@
+package com.mesosphere.usi.core.linting
+
+import com.mesosphere.usi.core.models.resources.{ExactValue, RangeRequirement, ResourceRequirement, ScalarRequirement}
+import com.mesosphere.usi.core.models.template.RunTemplate
+import com.mesosphere.usi.core.models.template.RunTemplate.KeyedResourceRequirement
+
+class Linting {
```

Comment:
As mentioned, this code was moved our of MesosEventLogic. I think I'd prefer to not do a lot with it right now, other than to move it, since much of it doesn't apply in the RunTemplate world.